hack/update-translations.sh

```diff
@@ -65,7 +65,7 @@ fi
 
 if [[ "${generate_pot}" == "true" ]]; then
   echo "Extracting strings to POT"
-  go-xgettext -k=i18n.T "${KUBECTL_FILES}" > tmp.pot
+  go-xgettext -k=i18n.T "${KUBECTL_FILES[@]}" > tmp.pot
```

Comment:
I think the -f flag above is broken by the way this script is written currently?

Reply:
Thank you for the comment.
I've updated the behavior with `-f` flag. By this update, we can run the script correctly with `-f foo.go bar.go` as the below(this is bash log).
```
$ hack/update-translations.sh -f "pkg/kubectl/cmd/cmd.go" -x
...
+ go-xgettext -k=i18n.T pkg/kubectl/cmd/cmd.go

$ hack/update-translations.sh -f "pkg/kubectl/cmd/cmd.go pkg/kubectl/cmd/profiling.go" -x
...
+ go-xgettext -k=i18n.T pkg/kubectl/cmd/cmd.go pkg/kubectl/cmd/profiling.go
```

However, if wild card character is contained, the script does not work.
```
$ hack/update-translations.sh -f "pkg/kubectl/cmd/*.go" -x
...
+ go-xgettext -k=i18n.T 'pkg/kubectl/cmd/*.go'
panic: open pkg/kubectl/cmd/*.go: no such file or directory
```

I have no idea how to avoid it reasonably. Should we avoid it?

Reply:
@BenTheElder Could you see this update when you have the time?